tests/test_extract_text.py

```diff
@@ -34,25 +39,56 @@ def mock_requests_get_image(*args, **kwargs):
         return response
 
 @patch('requests.get', side_effect=mock_requests_get_image)
-def test_extract_text_success(mock_get):
+def test_extract_text_success_with_openai(mock_get):
     url = "http://example.com"
     expected_images = ["http://example.com/image1.jpg", "http://example.com/image2.jpg"]
     expected_headlines = ["Mocked Ad Headline", "Mocked Ad Headline"]
 
     with patch("app.routes.OpenAI", return_value=mock_openai_client()):
         response = client.get("/extract-text", params={"url": url})
         assert response.status_code == 200
-        assert response.json() == {"images": expected_images, "headlines": expected_headlines}
-        for headline in response.json()["headlines"]:
+        actual_response = response.json()
+        print(f"Actual response: {actual_response}")
+        assert actual_response == {"images": expected_images, "headlines": expected_headlines}
+        for headline in actual_response["headlines"]:
             assert len(headline.split()) <= 5  # Ensure each headline is no more than 5 words
 
+@patch('requests.get', side_effect=mock_requests_get_image)
```

Comment:
Put this in a different file

Reply:
Thank you for the feedback, @lgaleana. I'll move the test to a separate file to keep the tests organized. I'll push the changes shortly.